OG/OG/AstVisiting/Visitor interfaces/Visitor interfaces/IFunctionNodeVisitor.cs

```diff
@@ -0,0 +1,10 @@
+﻿using OG.ASTBuilding.TreeNodes;
+
+namespace OG.AstVisiting
+{
+    public interface IFunctionNodeVisitor : IBodyNodeVisitor, IBoolFuncCallVisitor, IMathFuncCallVisitor, IPointFuncCallVisitor
+    {
+        public void Visit(IFunctionNode node);
+        public void Visit(FunctionNode node);
```

Comment:
end der brug for begge?

Reply:
JA, vi har flere function node typer for some reason

Reply:
Nej, du har ret